easybuild/easyconfigs/m/mkl_fft/mkl_fft-1.0.14-intel-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'mkl_fft'
+version = '1.0.14'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/IntelPython/mkl_fft'
+description = """NumPy-based Python interface to Intel(R) MKL FFT functionality"""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+toolchainopts = {'pic': True}
+
+dependencies = [
+    ('Python', '3.6.6'),
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+source_urls = ['https://github.com/IntelPython/mkl_fft/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['644ed4d845b741f80c6576c94bdbd14d18fce21e6cc725676e979b7f953154a8']
```

Comment:
Please move this part between `toolchain` and `dependencies`. It is not a strict error but we try to keep some common structure between the easyconfigs.